pg-index-health-logger/src/main/java/io/github/mfvanek/pg/common/maintenance/DatabaseChecks.java

```diff
@@ -74,11 +75,15 @@ public DatabaseChecks(@Nonnull final HighAvailabilityPgConnection haPgConnection
     public <T extends DbObject> DatabaseCheckOnCluster<T> getCheck(@Nonnull final Diagnostic diagnostic, @Nonnull final Class<T> type) {
         final DatabaseCheckOnCluster<?> check = checks.get(diagnostic);
         if (check == null) {
-            throw new IllegalStateException(String.format("Check for diagnostic %s not found", diagnostic));
+            throw new IllegalStateException(String.format(Locale.ROOT, "Check for diagnostic %s not found", diagnostic));
         }
         if (!type.isAssignableFrom(check.getType())) {
             throw new IllegalArgumentException("Illegal type: " + type);
         }
         return (DatabaseCheckOnCluster<T>) check;
     }
+
+    public void clearChecks() {
```

Comment:
Please remove this method

Reply:
The use of setAccessible in io.github.mfvanek.pg.support.TestUtils and io.github.mfvanek.pg.common.maintenance.DatabaseChecksTest causes an error in the foribdden-api check due to java-refelection. How should I handle this?

https://github.com/policeman-tools/forbidden-apis/issues/198
Currently forbidden-api doesn't seem to support excluding specific classes.

Reply:
> Should we try SuppressForbidden?

Yes, try to use https://github.com/policeman-tools/forbidden-apis/blob/main/src/main/java/de/thetaphi/forbiddenapis/SuppressForbidden.java